### ISSUE TITLE
some compilation flags are set to automatic instead of manual

### ISSUE THREAD

**Issue by chessai** [andrewthad/quickcheck-classes#78]
-f-semirings, -f-unary-laws, and -f-binary-laws are set to automatic (not manual). Is this an oversight? I'm almost certain that -f-semirings should be manual.

**Comment by skeuchel** [comment]
These are indented too much
https://github.com/andrewthad/quickcheck-classes/blob/5d9bdfccd8ca12357e6c11a2685c6fa833532011/quickcheck-classes.cabal#L49-L50
so that they became part of the description instead of configuring the flag. Maybe you can still update the metadata on hackage for this.

**Comment by chessai** [comment]
Oh, good catch.

**Comment by skeuchel** [comment]
Agreed, semirings should be manual. unary-laws and binary-laws are automatic by design at the moment, but they don't have to be automatic. I already commented https://github.com/andrewthad/quickcheck-classes/issues/54#issuecomment-445248762 on this in the CPP thread: 
> 1. Turn these automatic flags also into manual flags effectively using the lower bounds transformers>=0.5 and QuickCheck>=2.10 except for advanced users that can lower them.

It's a choice. I don't really feel strongly about it either way.

**Comment by andrewthad** [comment]
My preference is that all the flags on this be manual. They’re mostly there so that primitive’s test suite can build faster.
